float32 depth
float32 depth_calib
float32 depth_filt
float32 depth_der
